source/modules/overview/actions.rst

```diff
@@ -0,0 +1,178 @@
+#.  If the inventory object has a template management, choose the template to be applied ; 
+#.  Fill in the different fields of the empty form ;
+#.  Validate.
+
+An option located in user preferences, *After creation, go to created element*, allows to choose, after creating an object, if the newly created object is displayed or if a new empty form is displayed in order to create another object.
+
+Modification
+^^^^^^^^^^^^
+
+The permission to modify an object depend on defined permission in user profile.
+
+For all inventory objects, creation is realized in the same way:
+
+#.  Connect to GLPI ;
+#.  Go to the object page, for example, to modify a computer, go to `Assets > Computers` ;
+#.  Or alternatively: search the object in the search engine available on top of objects list ;
+#.  Two choices of modification are then possible:
+
+   * Unitary modification:
+
+      #.  Click on object name ;
+      #.  Modify fields ;
+      #.  Click on button **Update**.
+
+   * Mass modification:
+
+      #.  Select the checkbox located on the left of the object name ;
+      #.  Click on button **Actions** ;
+      #.  Choose field to be modified ;
+      #.  Enter new field value ;
+      #.  Click on button **Post**.
+
+Display
+^^^^^^^
+
+The permission to display an object depend on `read` permission in user profile.
+
+If the permission to display an object is not granted, the name of this object will not appear in GLPI different menus. For example, if the permission to read a `Computer` type object is not granted in the user's profile, the sub-menu `Computer` will not appear in menu `Assets`, even if the permission to modify a `Computer` is granted.
+
+For all inventory objects, displaying an object is realized in the same way:
+
+#. Connect to GLPI ;
+#. Go to the object page, for example, to display a computer, go to `Assets > Computers` ;
+#. Or alternatively: search the object in the search engine available on top of objects list ;
+#. Click on object name..
+
+
+Attaching a document
+^^^^^^^^^^^^^^^^^^^^
+
+When available for the item type, attaching a document is realized in the same way:
+
+#. Connect to GLPI ;
+#. Go to the object menu, for example, to attach to a computer, go to `Assets > Computers` ;
+#. Or alternatively: search the object in the search engine available on top of objects list ;
+
+Two possibilities are available:
+
+- Unitary attachment :
+
+   #. Click on the object name ;
+   #. Select tab **Documents** ;
+   #. Select a document or click button **Choose** to add a new document ;
+   #. Click on button **Add**.
+
+- Mass attachment :
+
+   #. Select the checkbox located on the left of the object name ;
+   #. Click on button **Actions** and choose **Add a Document**
+
+GLPI will then display the object form with newly attached document.
+
+
+Attaching a contract
+^^^^^^^^^^^^^^^^^^^^
+
+For all inventory objects, attaching a contract (if a contract can be attached to the object) is realized in the same way:
```

Comment:
Applies to more than just inventory types. Same as documents.

Reply:
I propose to change it to (similar to documents):

_When available for the item type, attaching a contract is realized in the same way:_

Reply:
Changed